pkg/templates/charts/toggle/console-mce/templates/console-quickstart.yaml

```diff
@@ -0,0 +1,136 @@
+    - group: operators.coreos.com
+      resource: operatorgroups
+      verb: list
+    - group: packages.operators.coreos.com
+      resource: packagemanifests
+      verb: list
+  description: 'With the multicluster switcher, you can easily switch between clusters'
+  displayName: Get started with multicluster switcher
+  conclusion: Congratulations! Multicluster switcher is ready to use.
+  durationMinutes: 15
+  icon: >-
+    data:image/svg+xml;base64,PHN2ZyBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCAxMDAgMTAwIiBoZWlnaHQ9IjEwMCIgdmlld0JveD0iMCAwIDEwMCAxMDAiIHdpZHRoPSIxMDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0ibTY2LjcgNTUuOGM2LjYgMCAxNi4xLTEuNCAxNi4xLTkuMiAwLS42IDAtMS4yLS4yLTEuOGwtMy45LTE3Yy0uOS0zLjctMS43LTUuNC04LjMtOC43LTUuMS0yLjYtMTYuMi02LjktMTkuNS02LjktMy4xIDAtNCA0LTcuNiA0LTMuNSAwLTYuMS0yLjktOS40LTIuOS0zLjIgMC01LjIgMi4xLTYuOCA2LjYgMCAwLTQuNCAxMi41LTUgMTQuMy0uMS4zLS4xLjctLjEgMSAuMSA0LjcgMTkuMiAyMC42IDQ0LjcgMjAuNm0xNy4xLTZjLjkgNC4zLjkgNC44LjkgNS4zIDAgNy40LTguMyAxMS40LTE5LjEgMTEuNC0yNC42IDAtNDYuMS0xNC40LTQ2LjEtMjMuOSAwLTEuMy4zLTIuNi44LTMuOS04LjkuNS0yMC4zIDIuMS0yMC4zIDEyLjIgMCAxNi41IDM5LjIgMzYuOSA3MC4yIDM2LjkgMjMuOCAwIDI5LjgtMTAuNyAyOS44LTE5LjIgMC02LjctNS44LTE0LjMtMTYuMi0xOC44IiBmaWxsPSIjZWQxYzI0Ii8+PHBhdGggZD0ibTgzLjggNDkuOGMuOSA0LjMuOSA0LjguOSA1LjMgMCA3LjQtOC4zIDExLjQtMTkuMSAxMS40LTI0LjYgMC00Ni4xLTE0LjQtNDYuMS0yMy45IDAtMS4zLjMtMi42LjgtMy45bDEuOS00LjhjLS4xLjMtLjEuNy0uMSAxIDAgNC44IDE5LjEgMjAuNyA0NC43IDIwLjcgNi42IDAgMTYuMS0xLjQgMTYuMS05LjIgMC0uNiAwLTEuMi0uMi0xLjh6IiBmaWxsPSIjMDEwMTAxIi8+PC9zdmc+
+  introduction: 'With the multicluster switcher, you can easily switch between clusters.'
+  tasks:
+    - description: >
+
+
+        To enable the console plug-in:
+
+        1. In the OpenShift Container Platform web console, switch to the
+        **[Administration]{{highlight qs-nav-administration}} > Cluster
+        settings**.
+
+
+        1. On the **Configuration** tab, click **Console
+        (operator.openshift.io)** 
+
+
+        1. On the **Console plugins** tab, check that the status of the `mce`
+        plug-in is **Enabled**.
+
+
+        1. If you are also using Red Hat Advanced Cluster Management for Kubernetes, check that
+        the status of the `acm` plug-in is also **Enabled**.
+
+
+        1. If the applicable plug-ins are not already enabled, for each one:
+          
+          
+          a. Click on the **Disabled** status. In the dialog that appears, select **Enable** and click on **Save**.  
+
+
+        After a few minutes, a popover with the text “Web console update is
+        available” appears. Click the **Refresh web console** link to load the web
+        console with the plug-ins enabled.
+      review:
+        failedTaskHelp: >-
+          This task is incomplete. Try the task again.
+        instructions: |-
+          Verify that the console plug-in was enabled successfully:
+
+            - In the navigation menu, is the first item a menu containing **All clusters** and **local-cluster**?
```

Comment:
I incorporated some of these changes awhile back but had to structure things a little differently because of the way the verification step is presented in a Quick Start.